### DISCUSSION TITLE
How can I make the fort icon always visible in the tray?

### DISCUSSION THREAD

**Discussion by b-user** [tnodir/fort#162] [Q&A]
How can I make the fort icon always visible in the tray (windows 7 x86) ?
I made many attempts to do this, but they were all unsuccessful: after closing the program or after rebooting the OS, the icon is always “hidden”.

**Comment by tnodir** [comment]
I can reproduce the problem. Need to investigate, why the icon is not always visible..

**Comment by b-user** [comment]
Okay, a little later I’ll try to describe it in more detail so that it can be reproduced.

**Reply by tnodir** [reply]
I can reproduce it easily. I do not understand, why the icon is hidden after reopening the app.
